packages/backend/src/apps/delay/helpers/generate-timestamp.ts

```diff
@@ -0,0 +1,27 @@
+import { DateTime } from 'luxon'
+
+const VALID_DATETIME_FORMATS = [
```

Comment:
If we're going to constrain formats, I'd rather we stick to only 1 supported format.

Reply:
Derp, ignore what I said above. I forgot this was due to formsg. 

very irritating..... thinking if we  can only support these 2 (formsg format, and existing format for backward compat) and nothing else.

Reply:
tbh these 3 formats are for backward compat + formsg after analysing the different inputs users put and ian thought its okay alsoo

Reply:
ok, in this case not a problem then, let's roll with these formats